src/applications/representative-appoint/tests/utilities/getApplicantName.unit.spec.jsx

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+import { getApplicantName } from '../../utilities/helpers';
+
+describe('getApplicantName', () => {
+  context('when the applicant is the Veteran', () => {
+    it('should Veteran name formatted', () => {
```

Comment:
Can you please give this a better description? This doesn't read right to me.

Reply:
Yep that's not correct